pgen-lib/build.gradle

```diff
@@ -0,0 +1,24 @@
+plugins {
+	id 'dev.nokee.cpp-library'
+	id 'cpp-unit-test'
+}
+
+ repositories {
+     // Use Maven Central for resolving dependencies.
+     mavenCentral()
+}
+
+library {
+    targetMachines = [machines.macOS.x86_64, machines.linux.x86_64]
```

Comment:
Neat!  Does it make both for you on command?

Reply:
Sadly, it does not. What it does appear to do is build the binary that is compatible with whatever system the build is running on, as long as it is in this list.

Reply:
Drats.  That's unfortunate.  I wonder if there is a way to make it crosscompile.

Reply:
Yeah, at some point I'm going to have to figure out how to do that....